sensu-plugins-chrony.gemspec

```diff
@@ -3,35 +3,44 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 require 'date'
 
-if RUBY_VERSION < '2.0.0'
-  require 'sensu-plugins-chrony'
-else
-  require_relative 'lib/sensu-plugins-chrony'
-end
+require_relative 'lib/sensu-plugins-chrony'
 
-Gem::Specification.new do |s|
-  s.authors                = ['Matteo Cerutti']
+Gem::Specification.new do |s| # rubocop:disable Metrics/BlockLength
+  s.authors = [
+    'Matteo Cerutti',
+    'Sensu-Plugins and contributors'
+  ]
   s.date                   = Date.today.to_s
   s.description            = 'This plugin provides facilities for monitoring Chrony NTP'
-  s.email                  = '<matteo.cerutti@hotmail.co.uk>'
+  s.email                  = [
+    '<matteo.cerutti@hotmail.co.uk>',
+    '<sensu-users@googlegroups.com>'
+  ]
   s.executables            = Dir.glob('bin/**/*').map { |file| File.basename(file) }
-  s.files                  = Dir.glob('{bin,lib}/**/*') + %w(LICENSE README.md CHANGELOG.md)
+  s.files                  = Dir.glob('{bin,lib}/**/*') + %w[LICENSE README.md CHANGELOG.md]
   s.homepage               = 'https://github.com/m4ce/sensu-plugins-chrony'
   s.license                = 'MIT'
   s.metadata               = { 'maintainer'         => '@m4ce',
                                'development_status' => 'active',
                                'production_status'  => 'stable',
                                'release_draft'      => 'false',
-                               'release_prerelease' => 'false'
-                              }
+                               'release_prerelease' => 'false' }
   s.name                   = 'sensu-plugins-chrony'
   s.platform               = Gem::Platform::RUBY
   s.post_install_message   = 'You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu'
   s.require_paths          = ['lib']
-  s.required_ruby_version  = '>= 1.9.3'
+  s.required_ruby_version  = '>= 2.0.0'
   s.summary                = 'Sensu plugins for monitoring Chrony NTP'
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsChrony::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin',   '1.2.0'
+  s.add_development_dependency 'bundler',                   '~> 1.7'
+  s.add_development_dependency 'github-markup',             '~> 1.3'
+  s.add_development_dependency 'rake',                      '~> 12.0'
+  s.add_development_dependency 'redcarpet',                 '~> 3.2'
+  s.add_development_dependency 'rspec',                     '~> 3.4'
+  s.add_development_dependency 'rubocop',                   '~> 0.49.0'
+  s.add_development_dependency 'yard',                      '~> 0.8'
+
+  s.add_runtime_dependency 'sensu-plugin', '1.2.0'
```

Comment:
Should not be locked to a single version. Should be `'~> 1.2'` or even `'~> 2.0'` with a major release.

Reply:
good call